ipatests/pytest_ipa/integration/create_bridge.py

```diff
@@ -0,0 +1,173 @@
+import re
+import textwrap
+
+from ipatests.pytest_ipa.integration import tasks
+
+
+def setup_scim_server(host, version="main"):
+    dir = "/opt/ipa-tuura"
+    password = host.config.admin_password
+    tasks.install_packages(host, ["unzip", "java-11-openjdk-headless",
+                                  "openssl", "maven", "wget", "git",
+                                  "firefox", "xorg-x11-server-Xvfb",
+                                  "python3-pip"])
+
+    # Download ipa-tuura project
+    url = "https://github.com/freeipa/ipa-tuura"
```

Comment:
nit: why not place dir, password and url altogether either declaring at the top of the fn or right before use?

Reply:
I declared dir and password at the top of the fn because they were used more than once throughout.  The others I thought were placed before their use.   Should I move dir and password to right before their first use maybe?

Reply:
That's a fair point, I'm ok with it as-is.